examples/meta/src/evaluation/cross_validation_classification.sg

```diff
@@ -0,0 +1,30 @@
+CSVFile f_feats("../../data/fm_train_real.dat")
+CSVFile f_labels("../../data/label_train_twoclass.dat")
+
+#![create_features]
+RealFeatures features(f_feats)
+BinaryLabels labels(f_labels)
+#![create_features]
+
+#![create_classifier]
+LibLinear svm()
+svm.set_liblinear_solver_type(enum LIBLINEAR_SOLVER_TYPE.L2R_L2LOSS_SVC)
+#![create_classifier]
+
+#![create_cross_validation]
```

Comment:
the comments are only necessary if you write a cookbook page (you can leave them for now though)